src/harness/harness.ts

```diff
@@ -808,10 +809,10 @@ module Harness {
             }
         }
 
-        export function createSourceFileAndAssertInvariants(fileName: string, sourceText: string, languageVersion: ts.ScriptTarget, assertInvariants = true) {
+        export function createSourceFileAndAssertInvariants(fileName: string, sourceText: string, languageVersion: ts.ScriptTarget, lightMode: boolean) {
             // Only set the parent nodes if we're asserting invariants.  We don't need them otherwise.

```

Comment:
We were just passing `!Harness.lightMode` everywhere anyway. I can change it back.